sgkit/io/plink/plink_reader.py

```diff
@@ -106,8 +106,10 @@ def coerce_code(v: dd.Series, codes: List[int]) -> dd.Series:
         v = v.where(v.isin(codes), np.nan)
         return v.fillna(-1).astype("int8")
 
-    df["paternal_id"] = df["paternal_id"].where(df["paternal_id"] != "0", None)
-    df["maternal_id"] = df["maternal_id"].where(df["maternal_id"] != "0", None)
+    # replace fam "0" with sgkit missing values (".")
```

Comment:
I think the missing value needs to be documented in the docstring of `read_plink`.